web/index.html

```diff
@@ -70,7 +70,7 @@
         <script src="https://polyfill.io/v3/polyfill.min.js?features=default%2CResizeObserver&flags=gated"></script>
     <% } %>
 </head>
-<body>
+<body id="bg">
```

Comment:
Probably don't need an id for this and could do something like:

```js
const body = document.querySelector('body');
```

Reply:
I tried that at first and am just now realizing I used the wrong selector, Thanks!